src/Microsoft.DotNet.ProjectModel/WorkspaceContext.cs

```diff
@@ -8,6 +8,7 @@
 using System.Linq;
```

Comment:
Yeah, this definitely makes WorkspaceContext a little gross. I think we're just going to have to live with that for RC2, we don't have time to do the good refactoring we want to do. Post-RC2, I have thoughts on a better way to structure these so that we can get a cleaner API.